spec/models/allocation_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe Allocation, type: :model do
```

Comment:
Empty spec?

Reply:
For now I guess the empty specs could just exercise the validation?

Reply:
Yeah I would have thought so

Reply:
Validation specs are on their way, just getting all the RSpec config sorted!